docs/reference/config_file.md

```diff
@@ -207,7 +207,7 @@ Each workspace within an environment is called a `component`
 In addition to the common workspace arguments, components also support the following arguments:
 
 - `eks` - Object that contains a `cluster_name` key to point terraform to a kubectl context
-- `kind` - String, either `terraform` or `helm_template` (DEPRECATED)
```

Comment:
is this still deprecated?

Reply:
yes (does nothing when empty or kind == terraform, fails when anything else)